README.md

```diff
@@ -116,31 +180,43 @@ For, Lisk Sepolia Testnet:
 
 ```sh
 ./bin/op-node \
-  --l1=$L1_RPC_URL \
-  --l1.rpckind=$L1_RPC_KIND \
-  --l1.beacon=$L1_BEACON_URL \
+  --l1=$OP_NODE_L1_ETH_RPC \
+  --l1.rpckind=$$OP_NODE_L1_RPC_KIND \
+  --l1.beacon=$OP_NODE_L1_BEACON \
   --l2=ws://localhost:8551 \
-  --l2.jwt-secret=./jwt.txt \
-  --network=lisk-sepolia \
-  --syncmode=execution-layer
+  --l2.jwt-secret=PATH_TO_JWT_TEXT_FILE \
+  --rollup.config=PATH_TO_NETWORK_ROLLUP_FILE
 ```
 
 For, Lisk Mainnet:
 
 ```sh
 ./bin/op-node \
-  --l1=$L1_RPC_URL \
-  --l1.rpckind=$L1_RPC_KIND \
-  --l1.beacon=$L1_BEACON_URL \
+  --l1=$OP_NODE_L1_ETH_RPC \
+  --l1.rpckind=$$OP_NODE_L1_RPC_KIND \
+  --l1.beacon=$OP_NODE_L1_BEACON \
   --l2=ws://localhost:8551 \
-  --l2.jwt-secret=./jwt.txt \
-  --network=lisk \
-  --syncmode=execution-layer
+  --l2.jwt-secret=PATH_TO_JWT_TEXT_FILE \
+  --rollup.config=PATH_TO_NETWORK_ROLLUP_FILE
 ```
 
+The above command start the `op-node` in the **full sync** mode. Depending on the chain length, the initial sync process could take significant time varying from days to weeks.
+The following logs indicate the initial process of `op-node` fetching sequentially blocks from Ethereum L1 to determine the OP Mainnet blocks that were published to Ethereum. During this process, `op-node` does not forward them to `op-geth`.
+
+```
+INFO [06-26|13:31:20.389] Advancing bq origin                      origin=17171d..1bc69b:8300332 originBehind=false
+```
+
+For more information refer to the OP [documentation](https://docs.optimism.io/builders/node-operators/tutorials/mainnet#full-sync).
+
+> **Note**:
+> <br>In case you skipped the step to initialize `op-geth` service mentioned above, you can start the node by adding`--network=OP_NODE_NETWORK` flag.
```

Comment:
```suggestion
> <br>In case you've skipped the `op-geth` [initialization step](#initialize-op-geth) above, you can start the node with the`--network=OP_NODE_NETWORK` flag.
```